python/lsst/ap/association/utils.py

```diff
@@ -134,6 +134,30 @@ def readSchemaFromApdb(apdb: Apdb) -> dict[str, schema_model.Table | None]:
     return {table.table_name(): apdb.tableDef(table) for table in ApdbTables}
 
 
+def checkSdmSchemaColumns(apdbSchema, colName, tableName):
+    """Check if a supplied column name exists in the schema.
+
+    Parameters
+    ----------
+    apdbSchema : `dict` [`str`, `lsst.dax.apdb.schema_model.Table`]
+        Schema from ``sdm_schemas`` containing the table definition to use.
+    colName : `str`
+        Name of the column to check for in the table.
+    tableName : `str`
+        Name of the table in the schema to use.
+
+    Returns
+    -------
+    `bool`
+        Is the specified column in the schema for the table?
+    """
+    table = apdbSchema[tableName]
```

Comment:
_might_ be faster to do something like

```
names = [columnDef.name for columnDef table.columns]
for col in colNames:
   if col not in names:
      Raise ValueError
```

where colNames is again a list of everything we're checking.